parsel/selector.py

```diff
@@ -334,7 +367,7 @@ def _get_root(
         text: str,
         base_url: Optional[str] = None,
         huge_tree: bool = LXML_SUPPORTS_HUGE_TREE,
-    ) -> Any:
+    ) -> etree._Element:
```

Comment:
I did not want to look into it when I recently had to face this in a different Parsel PR, but I wonder if we could stop using the private `_Element` class name without things working any differently. Looking at [this code](https://lxml.de/api/xml.etree.ElementTree-pysrc.html#L511) makes me think it may be possible.